.rubocop.yml

```diff
@@ -3,6 +3,8 @@ AllCops:
     - mixlib-install.gemspec
 Style/AlignParameters:
   EnforcedStyle: with_first_parameter
+Style/BlockDelimiters:
```

Comment:
I asked @mwrock to remove the `rubocop.yml` file in https://github.com/chef/mixlib-install/pull/136.